os/include/tinyara/sched.h

```diff
@@ -500,7 +494,7 @@ struct task_group_s {
 #endif
 
 #ifdef CONFIG_BINFMT_LOADABLE
-#define IS_LOADED_MODULE(group)    (group->tg_bininfo != NULL)   /* Points loading data if it is loaded */
+#define IS_BINARY_MAINTASK(tcb)    (tcb->tg_bininfo != NULL)   /* Points loading data if it is loaded */
```

Comment:
How about changing `IS_BINARY_MAINTASK(tcb)` to `IS_BINARY_MAINTASK(task_tcb)`? Because original tcb doesn't have `tg_bininfo`, so it can be confused.